docs/src/roots.md

```diff
@@ -366,15 +366,15 @@ dfᵏs (generic function with 1 method)
 
 ```jldoctest roots
 julia> Roots.newton(f, D(f), 2)
-2.0945514815423265
+3.141592653589793
```

Comment:
Hmm, have to see what happened here... That difference isn't a numerical one.

Reply:
Let me check again.

Reply:
You are right. By mistake I had changed the `f`